### PR TITLE
Update feature_column_v2.py

### PR DESCRIPTION
**Pull request by larrytin**
Or should be:
y_0 = 1.0 / 2.0 * ( w_a + w_ b) + w_d + b
y_1 = w_c + 1.0 / 3.0 * ( w_e + w_ f + w_g) + b

### CODE REVIEW

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @rohan100jain: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @rohan100jain: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 29 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @rohan100jain, @ymodak: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @rohan100jain, @ymodak: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @rohan100jain, @ymodak: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 29 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @rohan100jain: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

File: tensorflow/python/feature_column/feature_column_v2.py [hunk 1]
```diff
@@ -537,8 +537,8 @@ def __init__(self,
 
         with `sparse_combiner` as "mean", the linear model outputs conceptly are
         ```
-        y_0 = 1.0 / 2.0 * ( w_a + w_ b) + w_c + b_0
-        y_1 = w_d + 1.0 / 3.0 * ( w_e + w_ f + w_g) + b_1
+        y_0 = 1.0 / 3.0 * ( w_a + w_ b + w_c) + b_0
```
Comment (rohan100jain) [inline tensorflow/python/feature_column/feature_column_v2.py]: I think the formula is correct. We take the mean for the 0th row in the first case and the 1st row in the second case.

**Comment by rthadur** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!

**Pull request closed without merging**